plugins/web-worker-offloading/third-party/seo-by-rank-math.php

```diff
@@ -24,7 +24,10 @@ function plwwo_rank_math_configure( $configuration ): array {
 	$configuration = (array) $configuration;
 
 	$configuration['globalFns'][] = 'gtag'; // Because gtag() is defined in one script and called in another.
-	$configuration['forward'][]   = 'dataLayer.push'; // See <https://partytown.builder.io/forwarding-event>.
+
+	// Expose on the main tread. See <https://partytown.builder.io/forwarding-event>.
+	$configuration['forward'][] = 'dataLayer.push';
+	$configuration['forward'][] = 'gtag';
```

Comment:
The addition here means that scripts in the main thread will now be able to call `gtag()` whereas previously it was only available to other scripts offloaded to a worker.